httpbin/handlers.go

```diff
@@ -314,15 +314,26 @@ func (h *HTTPBin) Unstable(w http.ResponseWriter, r *http.Request) {
 // ResponseHeaders responds with a map of header values
 func (h *HTTPBin) ResponseHeaders(w http.ResponseWriter, r *http.Request) {
 	args := r.URL.Query()
+	body := make(map[string]interface{})
 	for k, vs := range args {
 		for _, v := range vs {
 			w.Header().Add(k, v)
 		}
+		if len(vs) == 1 {
+			body[k] = vs[0]
+		} else {
+			body[k] = vs
+		}
```

Comment:
Please drop this logic.  Per [my comment here](https://github.com/mccutchen/go-httpbin/issues/91#issuecomment-1667003449)

> But, to be clear, go-httpbin will continue to serialize header values (and form values and any other “zero or more” values inherent to HTTP) as JSON arrays for consistency, where original httpbin serializes them as single scalar values, arrays, or comma-separated strings, depending on the endpoint and incoming request parameters.

That might not be very clear.  To restate it, what I mean is that go-httpbin returns all header values as JSON arrays, even if only one header is given.

I know that is not actually backwards-compatible with the original httpbin, but for now I've decided that I prefer the consistency (and implementation simplicity) of this approach and I'm okay with the incompatibility.  (See https://github.com/mccutchen/go-httpbin/issues/37 and https://github.com/mccutchen/go-httpbin/issues/15 for related discussion.)